R/linter_tags.R

```diff
@@ -40,17 +42,20 @@
 #' identical(lintr_linters, lintr_linters2)
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
-available_linters <- function(packages = "lintr", tags = NULL) {
+available_linters <- function(packages = "lintr", tags = NULL, include_deprecated = FALSE) {
   if (!is.character(packages)) {
     stop("`packages` must be a character vector.")
   }
   if (!is.null(tags) && !is.character(tags)) {
     stop("`tags` must be a character vector.")
   }
+  if (!is.logical(include_deprecated) || length(include_deprecated) != 1L || is.na(include_deprecated)) {
+    stop("`include_deprecated` must be a flag.")
```

Comment:
I think `flag` is not a universal term, "logical scalar" or just "TRUE or FALSE" would be better to me